ringpop.go

```diff
@@ -96,6 +96,25 @@ func mergeDefault(opts *Options) *Options {
 	return opts
 }
 
+type Interface interface {
```

Comment:
I think I gave this suggestion. This It's used throughout the standard library, which is considered as excellent coding practice.

https://golang.org/pkg/container/heap/#Interface
https://golang.org/pkg/sort/#Interface

As a third party you see the type as ringpop.Interface which makes sense.

Admittedly I found this confusing when I came by it, but after a few moments it made sense to me. It also gives the third party a good hint that this is the most important thing in the package.